### ISSUE TITLE
Update np.array() calls on three separate arrays to single dask.compute() call

### ISSUE THREAD

**Issue by jni** [GenevieveBuckley/distributed-skeleton-analysis#1]
In the notebook/blog post, there is a cell where three different arrays (rows, cols, data) are created that potentially share tasks in the task graph. This could be sped up by using a single compute() call with all three arrays. Cell 25 here:

<img width="1302" alt="Screen Shot 2021-11-30 at 6 51 32 pm" src="https://user-images.githubusercontent.com/492549/144021939-87e8b252-2901-496f-afad-9d476abc5259.png">

The blog post should be updated not just with the "better" way, but with a comment showing the old way and explaining why the new way is better. Perfect real-world teachable moment! 😊

**Comment by GenevieveBuckley** [comment]
Yeah, I'm not going to be able to get around to this before I start my new job :cry: 

I agree with all of this. We should definitely do it ...sometime

**Comment by jni** [comment]
> We should definitely do it ...sometime

This is just par for the course in FOSS, I have learned to let it wash over me... :blush: :joy:
